test/sortitionPoolTest.js

```diff
@@ -1,156 +1,181 @@
-      owner,
+      owner.address,
     )
+    await pool.deployed()
   })
 
   describe("selectGroup", async () => {
-    it("returns group of expected size", async () => {
-      await staking.setStake(alice, 20000)
-      await staking.setStake(bob, 22000)
-      await staking.setStake(carol, 24000)
-      await pool.joinPool(alice)
-      await pool.joinPool(bob)
-      await pool.joinPool(carol)
-
-      const group = await pool.selectGroup(3, seed, {
-        from: owner,
+    context("when called by owner", () => {
+      beforeEach(async () => {
+        await staking.setStake(alice.address, 20000)
+        await staking.setStake(bob.address, 22000)
+        await staking.setStake(carol.address, 24000)
+        await pool.joinPool(alice.address)
+        await pool.joinPool(bob.address)
+        await pool.joinPool(carol.address)
       })
-      await pool.nonViewSelectGroup(3, seed, { from: owner })
-
-      assert.equal(group.length, 3)
-    })
 
-    it("reverts when called by non-owner", async () => {
-      await staking.setStake(alice, 20000)
-      await staking.setStake(bob, 22000)
-      await staking.setStake(carol, 24000)
-      await pool.joinPool(alice)
-      await pool.joinPool(bob)
-      await pool.joinPool(carol)
-
-      try {
-        await pool.selectGroup(3, seed, { from: accounts[0] })
-      } catch (error) {
-        assert.include(error.message, "Only owner may select groups")
-        return
-      }
-
-      assert.fail("Expected throw not received")
+      it("should return group of expected size", async () => {
+        const group = await pool.connect(owner).selectGroup(3, seed)
+        await pool.connect(owner).nonViewSelectGroup(3, seed)
+        expect(group.length).to.be.equal(3)
+      })
     })
 
-    it("reverts when there are no operators in pool", async () => {
-      try {
-        await pool.selectGroup(3, seed, { from: owner })
-      } catch (error) {
-        assert.include(error.message, "Not enough operators in pool")
-        return
-      }
+    context("when called by non-owner", () => {
+      beforeEach(async () => {
+        await staking.setStake(alice.address, 20000)
+        await staking.setStake(bob.address, 22000)
+        await staking.setStake(carol.address, 24000)
+        await pool.joinPool(alice.address)
+        await pool.joinPool(bob.address)
+        await pool.joinPool(carol.address)
+      })
 
-      assert.fail("Expected throw not received")
+      it("should revert", async () => {
+        await expect(
+          pool.connect(alice).selectGroup(3, seed),
+        ).to.be.revertedWith("Only owner may select groups")
+      })
     })
 
-    it("returns group of expected size if less operators are registered", async () => {
-      await staking.setStake(alice, 2000)
-      await pool.joinPool(alice)
-
-      const group = await pool.selectGroup(5, seed, {
-        from: owner,
+    context("reverts when there are no operators in pool", async () => {
```

Comment:
```
context("when there are no operators in the pool", async () => {
   it("should revert",
```

Reply:
Done in https://github.com/keep-network/sortition-pools/pull/127/commits/1fdcda0aae9ee9065e524e6280f2dc2157e2aaf3.